docs/404.html

```diff
@@ -19,12 +19,18 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="theme-color" content="#ffffff" />
 
+        
```

Comment:
@maldrake : what are these random newlines ?

Reply:
I have no explanation. These are auto-generated files, so it's whatever the book generator put into the files.  I could try regenerating the book to see if it's something transient, if they're problematic.

Reply:
Nah, it's doesn't really make a difference.